src/web/routes/application/send-code/notify.js

```diff
@@ -0,0 +1,29 @@
+const { NotifyClient } = require('notifications-node-client')
+const uuid = require('uuid/v4')
+const config = require('../../../../config')
+const notifyClient = new NotifyClient(config.environment.NOTIFY_API_KEY)
+require('dotenv').config()
+
+const { EMAIL, TEXT } = require('../common/constants')
+
+const EMAIL_TEMPLATE_ID = '18cf4f69-e1b5-4aa9-bed7-c906d3a59285'
+const SMS_TEMPLATE_ID = 'a770479a-097c-4079-acc2-d452a6f21585'
+
+function sendConfirmationCode (claim, channelForCode, confirmationCode) {
```

Comment:
Could the arguments for this just be
```
sendConfirmationCode (claim, channel, code)
```
as the extra words seem to be implied by the function name?

Reply:
Updated with simpler names.